url_test.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
```

Comment:
This is because the config vars on heroku, i wasn't able to setup them correctly. So I assumed some some Random values. will you help me in setting them up.